Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -469,7 +475,7 @@ public void AddBuff(Buff.BuffType buff)
 
         if (buff == Buff.BuffType.Bounce)
         {
-            QuadDamageTimer.Value = NetworkManager.ServerTime.TimeAsFloat + 10;
+            BounceTimer.Value = NetworkManager.ServerTime.TimeAsFloat + 10;
```

Comment:
What does this change solve / why is it needed?

Reply:
It is needed because the buff it relates to is the "bounce" buff, and therefore the bounceTimer needs to be increased not the quaddamageTimer. Through this change the correct effect of letting bullets bounce of objects e.g. walls is applied when collecting the buff in-game.